upgrade_test.py

```diff
@@ -404,7 +404,7 @@ def wait_for_node_to_finish():
                     r"find /var/lib/scylla/data/system -type f ! -path '*snapshots*' -printf %f\\n")
                 all_sstable_files = result.stdout.splitlines()
 
-                sstable_version_regex = re.compile(r'(\w+)-\d+-(.+)\.(db|txt|sha1|crc32)')
+                sstable_version_regex = re.compile(r'(\w+)-.+-(.+)\.(db|txt|sha1|crc32)')
```

Comment:
isn't . too broad? [\d\w]+ wouldn't be better?

Reply:
it's a uuid, an in general we don't care what's there, we care only on the sstable format version, which is the two letter at the beginning, I'm o.k. with it to be broad.